qt/icon.py

```diff
@@ -121,4 +121,5 @@
 INVALID_EXCLUDE     = QIcon.fromTheme('emblem-ohno',
                       QIcon.fromTheme('face-surprise'))
 
+ENCRYPT = QIcon.fromTheme('lock')
```

Comment:
`lock` is not contained in the (minimal) standard icon set of the [freedesktop.org spec](https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html).

This means this icon will only exist if it is contained in one of our [required icon schemes](https://github.com/bit-team/backintime/blob/dev/CONTRIBUTING.md#dependencies) which currently is only `oxygen` as **optional** (!!!) requirement.

Could you please check on your computer in which schemes the icon is contained?
In my newly installed computer I have found:

```
$ find /usr/share/icons -name lock.*
/usr/share/icons/Humanity/actions/16/lock.svg
/usr/share/icons/Humanity/actions/24/lock.svg
/usr/share/icons/Humanity/actions/48/lock.svg
/usr/share/icons/Humanity/actions/32/lock.svg
/usr/share/icons/breeze-dark/actions/16/lock.svg
/usr/share/icons/breeze-dark/actions/22/lock.svg
/usr/share/icons/breeze/actions/16/lock.svg
/usr/share/icons/breeze/actions/22/lock.svg
```

Perhaps we should use another icon from the standard set...